editor/scene_tree_editor.cpp

```diff
@@ -1359,6 +1359,10 @@ void SceneTreeDialog::_select() {
 	}
 }
 
+void SceneTreeDialog::_selected_changed() {
+	get_ok_button()->set_disabled(tree->get_selected() == nullptr);
```

Comment:
```suggestion
	get_ok_button()->set_disabled(!tree->get_selected());
```
I'd write it like that, but whichever style is ok.